tests/e2e/redis/basic-ingest-redis-serving.py

```diff
@@ -1091,48 +1095,59 @@ def core_service_stub(self, core_url):
         core_service_stub = CoreServiceStub(core_channel)
         return core_service_stub
 
-    def apply_feature_set(self, core_service_stub, feature_set_proto):
+    @pytest.fixture(scope="module")
+    def auth_meta_data(self, enable_auth):
```

Comment:
It's not clear to me why we need this method. Wouldn't this be handled automatically by the client library?

Reply:
`TestsBasedOnGrpc` class in the file is testing some of the functionality using grpc without using client.

Reply:
Hmm, this class is tech debt waiting to be moved back into client based tests. Is there a need to have GRPC only tests where client tests do not cover the test scenario ?

Reply:
Hmm, this is tech debt waiting to be moved back into to the client based tests. Not in scope for this PR.